azurecompute/src/main/java/org/jclouds/azurecompute/compute/options/AzureComputeTemplateOptions.java

```diff
@@ -54,6 +54,7 @@
    protected String networkSecurityGroupName;
    protected String reservedIPName;
    protected Boolean provisionGuestAgent;
+   protected Boolean winrmUseHttps;
```

Comment:
isn't `String listenerProtocol` a better model ? we can then add a `ListenerProtocol` enum like:
```
   public enum ListenerProtocol {

      HTTP("Http"),
      HTTPS("Https");

      private String value;

      ListenerProtocol(String value) {
         this.value = value;
      }

      public static ListenerProtocol fromString(String value) {
         ListenerProtocol[] items = ListenerProtocol.values();
         for (ListenerProtocol item : items) {
            if (item.toString().equalsIgnoreCase(value)) {
               return item;
            }
         }
         return new IllegalArgumentException("Cannot find an enum for " + value);
      }

      @Override
      public String toString() {
         return this.value;
      }
   }
```